cmd/deploy/deploy.go

```diff
@@ -589,10 +582,42 @@ func (dc *DeployCommand) deployStack(ctx context.Context, opts *Options) error {
 		Config:         cfg,
 		IsInsideDeploy: true,
 	}
-	oktetoLog.Information("Deploying compose")
 	return stackCommand.RunDeploy(ctx, composeSectionInfo.Stack, stackOpts)
 }
 
+func (dc *DeployCommand) deployDivert(ctx context.Context, opts *Options) error {
+	oktetoLog.SetStage("Divert configuration")
+	defer oktetoLog.SetStage("")
+
+	sp := utils.NewSpinner(fmt.Sprintf("Diverting namespace %s...", opts.Manifest.Deploy.Divert.Namespace))
+	sp.Start()
+	defer sp.Stop()
+
+	c, _, err := dc.K8sClientProvider.Provide(okteto.Context().Cfg)
+	if err != nil {
+		return err
+	}
+
+	result, err := c.NetworkingV1().Ingresses(opts.Manifest.Deploy.Divert.Namespace).List(ctx, metav1.ListOptions{})
```

Comment:
Are we supporting this for k8s clusters < 1.19?

Reply:
Maybe you can use https://github.com/okteto/okteto/blob/master/pkg/k8s/ingresses/crud.go#L97 to make it compatible with them

Reply:
We don't need to support <1.19 anymore.